src/node.ts

```diff
@@ -31,7 +31,8 @@ export enum InstanceType {
 export enum Network {
   MAINNET = 'n-ethereum-mainnet',
   ROPSTEN = 'n-ethereum-ropsten',
-  RINKEBY = 'n-ethereum-rinkeby'
+  RINKEBY = 'n-ethereum-rinkeby',
+  GOERLI = 'n-ethereum-goerli'
```

Comment:
Add a [trailing comma](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas) here so next time another network is added, the diff only shows one line changed.

Reply:
See commit [d20e6ec](https://github.com/cdklabs/cdk-ethereum-node/pull/67/commits/d20e6ec5a182339e3d495545722f8d5a134101bf)